content/blogposts/2022/openssl-307-vulnerabilities.md

```diff
@@ -124,30 +63,26 @@ However, the popular cryptography package does ship static OpenSSL libraries in
 (f:requirements.*txt cryptography(\s*[=~]=\s*(36\.|37\.|38\.0\.[0-2]))) OR (f:poetry.lock name\s*=\s*"cryptography" AND version\s*=\s*"(36\.|37\.|38\.0\.[0-2]))
 ```
 
-
 #### PHP
 
 Same general situation as Python. Users are unlikely to be affected, and if they are, a simple upgrade of the base image and rebuild should suffice.
 
-
-#### Go
-
-Go uses its own, completely independent SSL implementation. No action required.
-
-
 #### Rust
 
 Rust projects tend to use either OpenSSL or rustls to provide SSL/TLS functionality. Rustls users are unaffected.
 
 Most OpenSSL users will use the openssl crate in its default mode, which links to the system OpenSSL, which means that a normal image/binary rebuild will work for them. However, it's possible to configure the openssl crate to use an embedded OpenSSL, in which case [this query](https://sourcegraph.com/search?q=context:global+f:Cargo.lock+openssl-src+AND+version%5Cs*%3D%5Cs*%22300%5C.%5B0-9%5C.%5D%2B%5C%2B3%5C.0%5C.%5B0-6%5D&patternType=regexp) should reveal projects that need to be updated:
 
-
 ```regex
 f:Cargo.lock openssl-src AND version\s*=\s*"300\.[0-9\.]+\+3\.0\.[0-6]
 ```
 
+#### Go
+
+Go uses its own, completely independent SSL implementation. No action required.
+
 ## OpenSSL Checker
 
-You can also use our bookmarklet-based OpenSSL Checker tool available at [https://sourcegraph-community.github.io/openssl-checker/](https://sourcegraph-community.github.io/openssl-checker/) to check if your own projects are affected.
+You can use our bookmarklet-based OpenSSL Checker tool available at [https://sourcegraph-community.github.io/openssl-checker/](https://sourcegraph-community.github.io/openssl-checker/) to check if your own projects are affected.
```

Comment:
Waters down the takeaway, but is more accurate (_insert Futurama technically correct joke here_):

```suggestion
You can use our bookmarklet-based OpenSSL Checker tool available at [https://sourcegraph-community.github.io/openssl-checker/](https://sourcegraph-community.github.io/openssl-checker/) to check if your Rust and Python projects are affected.
```